multi_partner_learning.py

```diff
@@ -56,10 +57,11 @@ def __init__(self,
         self.minibatched_y_train = [None] * self.partners_count
         self.aggregation_weights = []
         self.models_weights_list = [None] * self.partners_count
+        self.federated_model_weights = weights_for_starting_model
         self.scores_last_learning_round = [None] * self.partners_count
         self.score_matrix_per_partner = np.nan * np.zeros(shape=(self.epoch_count, self.minibatch_count, self.partners_count))
         self.score_matrix_collective_models = np.nan * np.zeros(shape=(self.epoch_count, self.minibatch_count + 1))
-        self.loss_collective_models = []
+        self.loss_collective_models = [] 
```

Comment:
OK so for `multi_partner_learning.py` there seem to be a lot to adapt. I'd say:

- Adding a `loss_collective_models` table: seems needed, good addition
- Adding support for starting with an existing model: this has been done more in depth by @celinejacques in a recently merged PR. It would be best to remove this implementation, and adapt to what @celinejacques has done

Does that seem OK @Thomas-Galtier?

Reply:
Yep that seems ok